src/lib/infusion/src/components/inlineEdit/css/InlineEdit.css

```diff
@@ -0,0 +1,64 @@
+/* Themed Appearance + Themed States */
```

Comment:
Bumping this to say that now that `clean:build` is not run following an Infusion build, I've implemented the first two points above as part of the postinstall script. Infusion is now being sourced from NPM for this project and copied into the front end dependencies in the same was as the other modules sourced from NPM.